src/Components/Users/UserProfile.tsx

```diff
@@ -190,14 +190,17 @@ export default function UserProfile() {
   const handleValueChange = (phoneNo: any, name: string) => {
     if (phoneNo && parsePhoneNumberFromString(phoneNo)?.isPossible()) {
       const form = { ...states.form };
+      console.log("Phone is ", phoneNo);
       form[name] = phoneNo;
+      console.log("FOrm is ", form);
       dispatch({ type: "set_form", form });
     }
   };
```

Comment:
```suggestion
    if (phoneNo && parsePhoneNumberFromString(phoneNo)?.isPossible()) {
      const form = { ...states.form, name: phoneNo };
      dispatch({ type: "set_form", form });
    }
  };
```

Remove `console.log` here too

Reply:
All `console.log` removed